spec/trimmer_decorator_rspec.rb

```diff
@@ -0,0 +1,12 @@
+require '../trimmer_decorator'
+require '../person'
+
+describe TrimmerDecorator do
+  describe 'when the trimmer decorator class runs' do
+    it 'returns the correct must give us the first 9 characters of a name' do
+      person = Person.new(10, 'jose juan zepeda Romero')
+      decorator = TrimmerDecorator.new(person)
+      expect(decorator.correct_name).to eq 'jose juan '
+    end
+  end
```

Comment:
- [ ] Kindly delete the not required files for this project. And only include all the files for TDD.

Reply:
Hi @sja-thedude . First of all, we would like to thank for reviewing our project milestone and giving us through feedback. 
The above file you cited and others are unit testing files of the previous milestone of this project.  We created our branch, TDD on the base of 'dev' branch, where the dev branch currently has codes from the unit-test branch we did and merged yesterday. Therefore, we kindly leave this feedback.
Thanks